package.json

```diff
@@ -3,8 +3,7 @@
     "esbuild": "^0.8.5"
   },
   "scripts": {
-    "build-prod": "rm build/* && esbuild src/stylesheet.css --minify --outfile=build/stylesheet.css && cp src/upsell_panda.js build/upsell_panda.js && esbuild build/upsell_panda.js --minify --bundle --outfile=dist/upsell_panda.min.js --loader:.css=text --define:process.env.APP_URL=\"'www.upsellpanda.com'\"",
-    "build-dev": "rm build/* && esbuild src/stylesheet.css --minify --outfile=build/stylesheet.css && cp src/upsell_panda.js build/upsell_panda.js && esbuild build/upsell_panda.js --minify --bundle --outfile=build/upsell_panda.dev.js --loader:.css=text --define:process.env.APP_URL=\"'localhost:3000'\"",
-    "purge-cdn": "curl https://purge.jsdelivr.net/gh/Upsell-Panda/embed@master/dist/upsell_panda.min.js"
+    "build": "rm build/* && esbuild src/stylesheet.css --minify --outfile=build/stylesheet.css && cp src/upsell_panda.js build/upsell_panda.js && esbuild build/upsell_panda.js --minify --bundle --outfile=build/upsell_panda.dev.js --loader:.css=text --define:process.env.APP_URL=\"'localhost:3000'\" && esbuild build/upsell_panda.js --minify --bundle --outfile=dist/upsell_panda.min.js --loader:.css=text --define:process.env.APP_URL=\"'www.upsellpanda.com'\"",
```

Comment:
I'm assuming update to this to have `rm -f ...` will be included here after the other PR is merged

Reply:
Yep :+1: